<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(172, 76, 130);
background: qradialgradient(spread:pad, cx:0.41, cy:0.53, radius:0.482, fx:0.714689, fy:0.324, stop:0 rgba(234, 120, 163, 255), stop:0.443182 rgba(255, 110, 110, 255), stop:0.931818 rgba(255, 128, 128, 255))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>9</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color :rgba(250,180,255,80) ; border-style: none; border-radius: 10px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600; color:#00f4f8;&quot;&gt;cadastro de Wifu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="cname">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>83</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ;
font-size : 16px ;
 color:#fcf ; </string>
    </property>
    <property name="text">
     <string>Nome :</string>
    </property>
   </widget>
   <widget class="QLabel" name="csex">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>112</y>
      <width>44</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ;
font-size : 16px ;
 color:#fcf ; </string>
    </property>
    <property name="text">
     <string>Sexo :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ccategory">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>141</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ;
font-size : 16px ;
 color:#fcf ; </string>
    </property>
    <property name="text">
     <string>Categoria :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ctags">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>194</y>
      <width>43</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ;
font-size : 16px ;
 color:#fcf ; </string>
    </property>
    <property name="text">
     <string>Tags :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lname">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>84</y>
      <width>123</width>
      <height>20</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string extracomment="campo nome"/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #faf;  border-style: none;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ltags">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>196</y>
      <width>106</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #faf; border-style: none;</string>
    </property>
    <property name="maxLength">
     <number>400</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldescribe">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>224</y>
      <width>123</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="accessibleDescription">
     <string extracomment="descrição , anotaçoes ,caracteristicas especiais , bordões ou até  fazes pelo qual o personagem é conhecido"/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #faf; text-align :lef ; display: box;  border-style: none;</string>
    </property>
    <property name="inputMask">
     <string extracomment="descriçao do personagem"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>120</number>
    </property>
   </widget>
   <widget class="QLabel" name="ctags_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>223</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ;
font-size : 16px ;
 color:#fcf ; </string>
    </property>
    <property name="text">
     <string>Descrição :</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rf">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>114</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ; color: #fbf ; </string>
    </property>
    <property name="text">
     <string extracomment="sexo feminino">Feminino</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rm">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>114</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ; color: #fbf ; </string>
    </property>
    <property name="text">
     <string extracomment="sexo feminino">Masculino</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>74</y>
      <width>537</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none; </string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="cuploadImagen">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>252</y>
      <width>117</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="accessibleName">
     <string extracomment="upload imagem"/>
    </property>
    <property name="accessibleDescription">
     <string extracomment="upload da imagem"/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ;
font-size : 16px ;
 color:#fcf ; </string>
    </property>
    <property name="text">
     <string>Upload Imagen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bupload">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>252</y>
      <width>100</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e8f;
  border: none;
  color: white;
  padding: 10px 22px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 10px;
  cursor: pointer;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
    </layout>
   </widget>
   <widget class="QRadioButton" name="rcatAnime">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>143</y>
      <width>117</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ; color: #fbf ; </string>
    </property>
    <property name="text">
     <string extracomment="categoria anime / desenho">Anime,Desenho</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rcatSeries">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>143</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ; color: #fbf ; </string>
    </property>
    <property name="text">
     <string extracomment="categoria serie">Serie</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rcatFilme">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>143</y>
      <width>53</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ; color: #fbf ; </string>
    </property>
    <property name="text">
     <string extracomment="categoria filme">Filme</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rcatManga">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>143</y>
      <width>63</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ; color: #fbf ; </string>
    </property>
    <property name="text">
     <string extracomment="categoria manga">Manga</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rcatVtube">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>143</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ; color: #fbf ; </string>
    </property>
    <property name="text">
     <string extracomment="categoria vtube">Vtube</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rcatCantor">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>170</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ; color: #fbf ; </string>
    </property>
    <property name="text">
     <string extracomment="categoria cantor">Cantor(a)</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bsubmit">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>440</y>
      <width>83</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e8f;
  border: none;
  color: white;
  padding: 10px 22px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 13px;
   cursor: pointer;</string>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bverificar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>96</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e8f;
  border: none;
  color: white;
  padding: 10px 22px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 13px;
   cursor: pointer;</string>
    </property>
    <property name="text">
     <string>Verificar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rcatGame">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none ; color: #fbf ; </string>
    </property>
    <property name="text">
     <string extracomment="categoria game">Game</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
